scripts/build.ts

```diff
@@ -12,6 +12,17 @@ const options = {
   define: {
     'process.env.VERSION': `'${version}'`
   },
+  external: [
+    'node:process',
+    'jsr:@std/assert',
+    'jsr:@std/encoding/base64',
+    'jsr:@std/flags/',
+    'jsr:@std/fmt/colors',
+    'jsr:@std/fs/',
+    'jsr:@std/path/',
+    'jsr:@std/streams/',
+    'jsr:@std/io'
+  ],
```

Comment:
Hmm... I think we will need to double-check that this doesn't break the distributed binary by excluding these from the `build/main.ts` file...

Reply:
The `external` modules are required for the build to succeed due to JSR resolution issues.
The distributed binary works offline (`deno run build/main.js`) even after clearing the cache with `deno cache --reload src/deps.ts`, suggesting the modules are bundled rather than relying on cached runtime resolution.